core/probe.h

```diff
@@ -100,6 +104,13 @@ class GAMMARAY_CORE_EXPORT Probe : public QObject, public ProbeInterface
      */
     static bool isInitialized();
 
+#ifdef Q_OS_WIN
```

Comment:
I think I might be able to simplify things by directly unload after I called inject?

Reply:
hm no, that would crash